core/src/main/java/io/grpc/util/OutlierDetectionLoadBalancer.java

```diff
@@ -199,24 +199,7 @@ protected Helper delegate() {
     public Subchannel createSubchannel(CreateSubchannelArgs args) {
```

Comment:
The addresses are in CreateSubchannelArgs.

I'm not all that against using READY, but I think that approach makes more sense if tracking the currently-connected address instead of all possible addresses.

Reply:
I changed this to simply getting the addresses from the args.